mrpaulblack/tron
# [server] game hookup in server as well as some small fixes

## What does this PR do?

Further impl. of tron API spec on server. Also hookup of GameController in ServerController.

## Why is this change important?

Another step in impl. full support of tron API on backend.

## How to test this PR locally?

```gradle build```

``` gradle :server:run --args="-l trace -t 604800"```

You can test with a CLI client like `nc` for example (see doc in ServerController):

```nc -u 127.0.0.1 3000```

## Author's checklist

/

## Related issues
